kolibri/utils/cli.py

```diff
@@ -190,6 +200,31 @@ def _migrate_databases():
     call_command("loaddata", "scopedefinitions")
 
 
+def _automatic_provisiondevice():
+    try:
+        with open(provisiondevice_file, "r") as f:
+            logging.info("Running initial setup from 'provisiondevice.json'")
+            options = json.load(f)
+    except ValueError as e:
+        logging.error(
+            "Attempted to load 'provisiondevice.json' but failed to parse JSON:\n{}".format(
+                e
+            )
+        )
+        sys.exit(1)
+    except FileNotFoundError:
+        options = None
+
+    if isinstance(options, Mapping):
+        options.setdefault("superusername", None)
+        options.setdefault("superuserpassword", None)
+        options.setdefault("preset", "nonformal")
+        options.setdefault("language_id", None)
+        options.setdefault("facility_settings", {})
+        options.setdefault("device_settings", {})
+        call_command("provisiondevice", interactive=False, **options)
```

Comment:
maybe `provisiondevice.json` should be deleted after success, because presumably it becomes a "dead" file after provisioning?

Reply:
Sure! I added that :)